news/2022-10-21-project-loved-october-2022.md

```diff
@@ -0,0 +1,374 @@
+
+Here we go again. Do you remember [Soulless 4](https://osu.ppy.sh/beatmapsets/772138#fruits/1623653), that one menacing map by [Kyptoric](https://osu.ppy.sh/users/1361268) that has only a single FC despite being loved for years already? This is pretty much its younger brother. In terms of patterns and overall feel of the map Soulless 3 is very similar to Soulless 4, so if you know about Soulless 4 you know what to expect from this one. The major difference though is that Soulless 3 is less than half the length of Soulless 4 which will probably make it more doable than its predecessor. But even then, this is a ridiculous challenge for just about anyone.
+
+[![](/wiki/shared/news/2022-10-21-project-loved-october-2022/1546722.jpg)](https://osu.ppy.sh/community/forums/topics/{{TOPIC_ID}})
+
+[Seiryu - AO-Infinity](https://osu.ppy.sh/beatmapsets/1546722#fruits) by [Saikai](https://osu.ppy.sh/users/8172283)\
+191 BPM, 3:30 | 7.22★\
+*description written by [Tenshichan](https://osu.ppy.sh/users/1101600)*
+
+[Saikai](https://osu.ppy.sh/users/8172283) is known for being an insane player who is currently top 10 on the global leader boards, but apart from all the impressive scores they set regularly, they also do map quite a lot.
+
+One of those maps is *AO-Infinity*, which is overall a clean map with some unconventional patterning here and there. A major part of this map are stream patterns with lots of hyperdashes, but there are also some parts that may throw you off, especially the middle section of the map since it contains some uncommon hyperdashes paired with fast-paced patterns. The most difficult section though is near the end of the map, where the stream patterns get noticeably more difficult and also contain quite the amount of fast wiggles in between. If you are not careful enough, a slip up may end the run there.
+
+Overall a map of moderate difficulty that can be enjoyed by a lot of people so make sure to check this map out!
+
+[![](/wiki/shared/news/2022-10-21-project-loved-october-2022/184710.jpg)](https://osu.ppy.sh/community/forums/topics/{{TOPIC_ID}})
+
+[Sayaka Sasaki - Zzz](https://osu.ppy.sh/beatmapsets/184710#fruits) by [Spectator](https://osu.ppy.sh/users/702598), [CLSW](https://osu.ppy.sh/users/531253) and [ZHSteven](https://osu.ppy.sh/users/142413)\
+154 BPM, 4:04 | 2.94★, 3.62★, 4.39★\
+*description written by [Sartan](https://osu.ppy.sh/users/4100941)*
+
+......................................huh- OH HEY THERE LOVELY READER! You scared me there! What was I doing? Well, you know, there is this map called Zzz by [Spectator](https://osu.ppy.sh/users/702598), and it's like the most soothing thing ever! It has a couple of difficulties which are very relaxing, and playable by virtually anyone. As a matter of fact, I believe it might be quite a good pick for loved, considering the amount of times it has been played since it's creation over eight years ago; it even saw some play in tournaments!
```

Comment:
```suggestion
......................................huh- OH HEY THERE LOVELY READER! You scared me there! What was I doing? Well, you know, there is this map called *Zzz* by [Spectator](https://osu.ppy.sh/users/702598), and it's like the most soothing thing ever! It has a couple of difficulties which are very relaxing, and playable by virtually anyone. As a matter of fact, I believe it might be quite a good pick for loved, considering the amount of times it has been played since its creation over eight years ago — it even saw some play in tournaments!
```